### DISCUSSION TITLE
Additional custom CSS

### DISCUSSION THREAD

**Discussion by kagiura** [retypeapp/retype#159] [Q&A] [answered]
Hello, new user of Retype, this is an awesome project! Thank you so much

Out of curiosity, is it possible right now (or planned) to add in custom CSS to the rendered pages? This could also be a temporary solution as more detailed appearance customization aren't available yet. (Mainly looking at fonts and colors here)

**Comment by fabriciomurta** [comment, marked as answer]
Glad you're enjoying Retype, Yuuki!

We do have plans on allowing for customization of styling (#57, #81, & probably more), but unfortunately, to this point there are no means to customize CSS in Retype. Please take a look over the conversations so you get a bit more insight on what's going on.

Thanks for your interest in Retype!

**Comment by alisman** [comment]
@yuuukun FYI, you can add <style></style> tag any page and put custom styles there.  not great (and not global) but works

**Reply by geoffreymcgill** [reply]
@alisman & @alisman You can use the new `/_includes/head.html` to add a global `.css` file. See the following comment for more details:

https://github.com/retypeapp/retype/discussions/282#discussioncomment-2343005

I'm actually working on the documentation for this feature today, so I'll post another update once those docs are published.
